crates/red_knot/src/db.rs

```diff
@@ -40,6 +43,7 @@ pub struct SourceJar {
 #[derive(Debug, Default)]
 pub struct SemanticJar {
     pub module_resolver: ModuleResolver,
+    pub symbol_tables: SymbolsTableStorage,
```

Comment:
Here we call the same thing both "a symbols table" and "symbol tables" in a single line of code, where I'd been calling it just "a symbol table" :)

I think frequently "a symbol table" is used to refer to the symbols in a single scope, so a module would be considered to have a tree of "symbol tables". So perhaps we should change naming to call the per-module thing `SymbolTables` instead, that might be most consistent with common usage. But I think it's also fine to call it `SymbolTable`, as I had done.

I've never seen it called a "symbols table".

In any case, we should decide whether a module has "a symbol table" or "symbol tables" (I'm fine with either one) or a "symbols table" (I'm opposed to this one), and be consistent with our naming everywhere.

Reply:
(Oh, actually the name `symbol_tables` for this field makes perfect sense either way, as it's storing symbol tables for multiple modules.)